renku/core/models/datasets.py

```diff
@@ -262,6 +262,8 @@ class DatasetFile(Entity, CreatorsMixin):
 
     url = jsonld.ib(default=None, context='schema:url', kw_only=True)
 
+    source = jsonld.ib(default=None, context='schema:isBasedOn', kw_only=True)
```

Comment:
We should probably add a small comment to help differentiate and avoid possible future confusions with `__source__`.

Reply:
I've changed this to `basedOn`